### DISCUSSION TITLE
Registration

### DISCUSSION THREAD

**Discussion by kitkatr** [IBAHCM/RPiR#63] [Archive]
kat hopkinson

**Comment by richardreeve** [comment]
Okay, I've sent you an invitation - let me know if you have any problems.

**Comment by richardreeve** [comment]
@kitkatr In the future, please do make sure you put any discussion in the right category - this was in *General* and should have been in *Registration*.

**Comment by richardreeve** [comment]
I've moved this into Registration btw.
